e2e/tests/stablex_test.rs

```diff
@@ -80,6 +86,21 @@ fn test_with_ganache() {
         .send()
         .wait()
         .expect("Cannot place first order");
+
+    // This order will not be matched
```

Comment:
I wouldn't add this here. This is a very specific scenario that is being tested and we want to keep the overall e2e test as general as possible (and not add every regression). If this one had been properly in place it should have actually already caught this issue as rinkeby had a bunch of orders in it that would have produced a bad output json.